cmake/Config.cmake.in

```diff
@@ -2,13 +2,13 @@
 
 include(CMakeFindDependencyMacro)
 
-find_dependency(OpenCV REQUIRED)
-find_dependency(Boost CONFIG REQUIRED)
-find_dependency(Eigen3 CONFIG REQUIRED)
-find_dependency(cereal CONFIG REQUIRED)
-find_dependency(glm REQUIRED)
-find_dependency(nanoflann CONFIG REQUIRED)
-find_dependency(eigen3-nnls CONFIG REQUIRED)
+find_dependency(OpenCV)
+find_dependency(Boost CONFIG system)
```

Comment:
You are right.  `COMPONENTS` isn't forwarded:
* https://cmake.org/cmake/help/v3.8/module/CMakeFindDependencyMacro.html
* https://cmake.org/pipermail/cmake-developers/2016-February/027876.html

The local jekins.py test confirms this.  I will replace that one with find_package as follows:

`find_dependency(Boost CONFIG system)` -> `find_package(Boost CONFIG REQUIRED system)`